schema_utils_test.py

```diff
@@ -433,6 +433,55 @@ def test_schemas_are_correctly_validated(self):
             with self.assertRaises((AssertionError, KeyError)):
                 validate_schema(schema)
 
+    def test_is_regex_raises_not_implemented_error(self):
```

Comment:
If these two functions are not used anywhere in the codebase, let's just get rid of them from schema_utils.py.

Reply:
removed in next commit